bindings/python/examples/test_torus_knot.py

```diff
@@ -81,9 +81,9 @@ def func_single(v):
         return pts[0]
 
     if warp_single:
-        return circle.revolve(int(m)).warp(func_single)
+        return Manifold.revolve(circle, int(m)).warp(func_single)
     else:
-        return circle.revolve(int(m)).warp_batch(func)
+        return Manifold.revolve(circle, int(m)).warp_batch(func)
```

Comment:
Maybe leave one example using `circle.extrude()` for variety? Unless there's already one that you didn't change.

Reply:
I think if we are having it in the `all_apis.py` it should be fine.